helm/alfresco-content-services/tests/repository_test.yaml

```diff
@@ -0,0 +1,30 @@
+---
+suite: test repository manifest
+templates:
+  - deployment-repository.yaml
+  - config-repository.yaml
+  - secret-database.yaml
+  - secret-s3.yaml
+  - config-dev-log4j-properties.yaml
+tests:
+  - it: should have basic metadata in place
+    values: &testvalues
+      - values/test_values.yaml
+    asserts:
+      - equal:
```

Comment:
please specify the actual template to be asserted in each assertion, as defaults seem to change with plugin implementation